src/Model/PushSubscriber.php

```diff
@@ -24,7 +24,10 @@ public static function publishFeed($priority = PRIORITY_HIGH)
 			// We always handle retries with low priority
 			if ($subscriber["push"] > 1) {
 				$priority = PRIORITY_LOW;
+			} else {
+				$priority = $default_priority;
```

Comment:
What was the issue?

Reply:
It is partly related to issue https://github.com/friendica/friendica/issues/5043. While testing the fix I realised that it took really, really long until my test posts came through.

Reply:
I was asking specifically about the add of the `$default_priority` variable in commit https://github.com/friendica/friendica/pull/5077/commits/435501449213958b41d3dbeeacafb4f534aef374

Reply:
This answer does fit as well :-) "remote-self" contact related push subscriptions are handled with a lower priority - like the delivery.

Reply:
No but you simply used `$priority` before, why did you need to add an intermediate variable on top of it? This is a code question, not a feature one.

Reply:
It is in a loop. So we have to reset it with every loop, since it depends on the "push" level.

Reply:
Ha, I see it now, thanks for the explanation!

Reply:
BTW: I stumbled upon it as well.